### PR TITLE
Update ghost-basics tests for new versions

### PR DESCRIPTION
**Pull request by acburdine**
refs https://github.com/docker-library/ghost/pull/194
- use correct api endpoints for different versions of Ghost

### CODE REVIEW

File: test/tests/ghost-basics/run.sh [hunk 1]
```diff
@@ -30,6 +30,8 @@ _request() {
 ghostVersion="$(docker inspect --format '{{range .Config.Env}}{{ . }}{{"\n"}}{{end}}' "$serverImage" | awk -F= '$1 == "GHOST_VERSION" { print $2 }')"
 case "$ghostVersion" in
 	0.*) _request GET '/ghost/' -I |tac|tac| grep -q '^Location: .*setup' ;;
-	*)   _request GET '/ghost/api/v0.1/authentication/setup/' |tac|tac| grep -q 'status":false' ;;
+	1.*)   _request GET '/ghost/api/v0.1/authentication/setup/' |tac|tac| grep -q 'status":false' ;;
+	2.*)   _request GET '/ghost/api/v2/admin/authentication/setup/' |tac|tac| grep -q 'status":false' ;;
+	3.*)   _request GET '/ghost/api/v3/admin/authentication/setup/' |tac|tac| grep -q 'status":false' ;;
```
Comment (tianon) [inline test/tests/ghost-basics/run.sh]: I'm a fan of the approach here, but I think we should still have some kind of fallback, even if it's just explicit failure (`false` or `exit 1`) to remind us at 4.x that something needs to be done here. :sweat_smile:

(Also the spacing after the `)` looks kind of odd -- was there a reason for the triple-space there?)

Comment (acburdine) [reply test/tests/ghost-basics/run.sh]: makes sense, I'll fix that

Comment (acburdine) [reply test/tests/ghost-basics/run.sh]: @tianon this should be fixed now 😄

**Comment by tianon** [comment]
LGTM

Build test of #6837; a7d55dd0; `amd64` (`ghost`):

```console

$ bashbrew build ghost:2.37.0
Building bashbrew/cache:0cf1148a25e026442c4b4f2d8cf383b6f2c568e58b6ef0df0cd642b4114a9e16 (ghost:2.37.0)
Tagging ghost:2.37.0
Tagging ghost:2.37
Tagging ghost:2
Tagging ghost:latest

$ test/run.sh ghost:2.37.0
testing ghost:2.37.0
	'utc' [1/5]...passed
	'cve-2014--shellshock' [2/5]...passed
	'no-hard-coded-passwords' [3/5]...passed
	'override-cmd' [4/5]...passed
	'ghost-basics' [5/5].....passed


$ bashbrew build ghost:2.37.0-alpine
Using bashbrew/cache:bc5caa023d4bcc3704fab9ef8556def38e105ab616c8f3599b534ee3772c8f61 (ghost:2.37.0-alpine)
Tagging ghost:2.37.0-alpine
Tagging ghost:2.37-alpine
Tagging ghost:2-alpine
Tagging ghost:alpine

$ test/run.sh ghost:2.37.0-alpine
testing ghost:2.37.0-alpine
	'utc' [1/5]...passed
	'cve-2014--shellshock' [2/5]...passed
	'no-hard-coded-passwords' [3/5]...passed
	'override-cmd' [4/5]...passed
	'ghost-basics' [5/5]......passed


$ bashbrew build ghost:1.26.0
Building bashbrew/cache:ca452000734185406148a4d326d9d89fc97cbde3d79e53ae3ceaf465789ab231 (ghost:1.26.0)
Tagging ghost:1.26.0
Tagging ghost:1.26
Tagging ghost:1

$ test/run.sh ghost:1.26.0
testing ghost:1.26.0
	'utc' [1/5]...passed
	'cve-2014--shellshock' [2/5]...passed
	'no-hard-coded-passwords' [3/5]...passed
	'override-cmd' [4/5]...passed
	'ghost-basics' [5/5].....passed


$ bashbrew build ghost:1.26.0-alpine
Using bashbrew/cache:3ad54ae625beb1578c7867a50ed5421970ed5da3d8b465259b41e07f1636a3e6 (ghost:1.26.0-alpine)
Tagging ghost:1.26.0-alpine
Tagging ghost:1.26-alpine
Tagging ghost:1-alpine

$ test/run.sh ghost:1.26.0-alpine
testing ghost:1.26.0-alpine
	'utc' [1/5]...passed
	'cve-2014--shellshock' [2/5]...passed
	'no-hard-coded-passwords' [3/5]...passed
	'override-cmd' [4/5]...passed
	'ghost-basics' [5/5].....passed

```

**Pull request merged**